### PR TITLE
[12.0][MIG] product_profile_example

### PR DESCRIPTION
**Pull request by kevinkhao**
depends on #546

### CODE REVIEW

File: product_profile_example/__manifest__.py [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-    ],
-    'installable': True,
-    'license': 'AGPL-3',
+    "name": "Product Profile Example",
+    "summary": "Product Profile Use Case",
+    "version": "12.0.1.0.1",
+    "author": "Akretion, Odoo Community Association (OCA)",
+    "category": "product",
+    "depends": ["product_profile", "purchase_stock", "point_of_sale", "mrp"],
+    "website": "http://www.akretion.com/",
```
Comment (nikul-serpentcs) [inline product_profile_example/__manifest__.py]: @kevinkhao https://github.com/OCA/product-attribute

File: product_profile_example/__manifest__.py [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
-    'website': 'http://www.akretion.com/',
-    'demo': [
-        'product.profile.csv',
-        'product_product_demo.xml',
-    ],
-    'installable': True,
-    'license': 'AGPL-3',
+    "name": "Product Profile Example",
+    "summary": "Product Profile Use Case",
+    "version": "12.0.1.0.1",
```
Comment (nikul-serpentcs) [inline product_profile_example/__manifest__.py]: `12.0.1.0.0`

**Review by nikul-serpentcs — COMMENTED** [review]
nitpicking

File: product_profile_example/__manifest__.py (hunk 2 above)
Comment (kevinkhao) [reply product_profile_example/__manifest__.py]: Hi Nikul, removing the license triggers the pylint error:
[C8102(manifest-required-key), ] Missing required key "license" in manifest file
or is there something that I'm misunderstanding ?

**Comment by EBII** [comment]
+1

File: product_profile_example/__manifest__.py (hunk 2 above)
Comment (bealdav) [reply product_profile_example/__manifest__.py]: I think nikul want you switch from 12.0.1.0.1 to 12.0.1.0.0

File: product_profile_example/__manifest__.py (hunk 1 above)
Comment (bealdav) [reply product_profile_example/__manifest__.py]: please switch to repo link instead of akretion link

**Review by bealdav — CHANGES_REQUESTED** [review]
Thanks Kevin

**Comment by ypapouin** [comment]
@kevinkhao, can you rebase and force push to pass test since product_profile is now merged ?

**Pull request closed without merging**